packages/libraries/main/src/utils/mergeUserInputDictionary.ts

```diff
@@ -0,0 +1,44 @@
+import { RankedDictionaries, UserInputsOptions } from '../types'
+
+export default (
+  optionsRankedDictionaries: RankedDictionaries,
+  optionsRankedDictionariesMaxWordSize: Record<string, number>,
+  userInputsOptions?: UserInputsOptions,
+) => {
+  const rankedDictionaries = {
+    ...optionsRankedDictionaries,
+  }
+  const rankedDictionariesMaxWordSize = {
+    ...optionsRankedDictionariesMaxWordSize,
+  }
+  if (!userInputsOptions) {
+    return {
+      rankedDictionaries,
+      rankedDictionariesMaxWordSize,
+    }
+  }
+
+  if (rankedDictionaries.userInputs) {
+    const isWordSizeBigger =
+      rankedDictionariesMaxWordSize.userInputs <
+      userInputsOptions.rankedDictionaryMaxWordSize
+
+    rankedDictionaries.userInputs = {
+      ...rankedDictionaries.userInputs,
+      ...userInputsOptions.rankedDictionary,
+    }
+
+    rankedDictionariesMaxWordSize.userInputs = isWordSizeBigger
```

Comment:
Couldn't this just be `Math.max(userInputsOptions.rankedDictionaryMaxWordSize, rankedDictionariesMaxWordSize.userInputs)`?

Then you wouldn't need to calculate `isWordSizeBigger` either.

Reply:
Thank you, i always forget that `Math.max` is a thing 👍